test.md

```diff
@@ -270,6 +270,38 @@ print it and run it through the `markdownify` Liquid filter. Here's a demo:
 </div>
 </div>
 
+#### Synchronizing multiple tab groups
```

Comment:
No, I think using `data-group` is better because that's what it is intended for. Later on we will have some more universal data groups that users will be able to set cookies to prefer (like always show me python if it's available).